### ISSUE TITLE
labels need an author

### ISSUE THREAD

**Issue by cmosig** [Deadwood-ai/deadtrees#61]
We have cases where imagery is provided by one institution and labels by another. There will more such cases in the future, especially when we start to generate labels for user-submitted orthophotos. 

We should probably address this before we make deadtrees.earth the primary copy of the database. 

The change should not  be complicated, it is only an additional column in the labels table and we need to have it appear on the website.

**Comment by mmaelicke** [comment]
Just as a comment: The api and dB already allow for different authors here. Should therefore be a frontend Isabel

**Comment by cmosig** [comment]
But the table in supabase does not contain an author?

![image](https://github.com/user-attachments/assets/c6d927b3-2b09-4504-95c9-b1ecd10ccc6d)

Author is different than user-id.

**Comment by mmaelicke** [comment]
Ah got you. So you upload a Label, which is not owned by you and the owner might even not be registered.

**Comment by cmosig** [comment]
Yes same with the datasets, where you have a user-id, but then also a `authors` string. 

![image](https://github.com/user-attachments/assets/17e2c669-e2f2-4a9a-ad77-db98b2c07fcc)
